SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduledJobPoller.java

```diff
@@ -112,10 +112,20 @@ public void runActionOnPoll() {
 
       final CronExpression cronExpression;
 
+      Optional<String> maybeSchedule = request.getRequest().getQuartzScheduleOrSchedule();
+      if (!maybeSchedule.isPresent()) {
```

Comment:
Deleted the other comment because it was simulposted with one from @wsorenson
I am happy to close this PR and move discussion somewhere else if everyone else is satisfied for now, but it doesn't appear that is the case.